model/service.go

```diff
@@ -107,6 +112,23 @@ const (
 	ProtocolRedis Protocol = "Redis"
 )
 
+// AuthenticationPolicy defines authentication policy for port.
+type AuthenticationPolicy string
```

Comment:
Suggestion: extend AuthPolicy proto definition https://github.com/istio/api/blob/master/proxy/v1/config/proxy_mesh.proto#L21 with a default value. Make the default be the default. And then explain the overriding principle in the API proto definition: per-service settings takes precedence over the per-mesh.

Reply:
Do you suggest extend API enum and use it here? Do we want to do it now or it can wait?

Reply:
I think a separate PR would be far better, no need to make this too complex.